psydac/api/tests/test_2d_multipatch_poisson.py

```diff
@@ -177,9 +178,10 @@ def test_poisson_2d_2_patch_dirichlet_4():
 
     D1 = M1(A)
     D2 = M2(B)
-    domain = D1.join(D2, name = 'domain',
-                bnd_minus = D1.get_boundary(axis=0, ext=-1),
-                bnd_plus  = D2.get_boundary(axis=0, ext=-1))
+
+    connectivity = [((0,0,-1),(1,0,-1))]
+    domains = [D1,D2]
+    domain = Domain.join(domains, connectivity, 'domain')
```

Comment:
As above.

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)